### PR TITLE
initial cleanup and generalization for all ERC-20 transactions

### PR DESCRIPTION
**Pull request by arutun**
This is the first step in supporting multiple ERC-20 tokens in a simple fashion.

It also cleans up some of the overlap between Eth transactions and any generic ERC-20 token, eliminating duplicate code.

### CODE REVIEW

File: packages/lndr/erc20-utils/index.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import ERC20Transaction from 'lndr/erc20-transaction'
+import Tx from 'ethereumjs-tx'
+import Web3 from 'web3'
+
+export const WEI_PER_ETH = Math.pow(10, 18)
+
+// supported tokens
+export const ERC20_BCPT = 'BCPT'
+
+const ERC20_Tokens = [
```
Comment (willbach) [inline packages/lndr/erc20-utils/index.ts]: this needs an `export` in front of it

File: packages/ui/dialogs/my-account/index.tsx [hunk 2]
```diff
@@ -372,6 +373,30 @@ class MyAccount extends Component<Props, State> {
     )
   }
 
+  renderCryptoBalancesSubpanel() {
+    const { ethBalance, bcptBalance } = this.props.state
+
+    const cryptoSubpanels = ERC20_Tokens.map( (token) => {
+      return (
+        <View style={style.spaceHorizontalL}>
```
Comment (willbach) [inline packages/ui/dialogs/my-account/index.tsx]: the top-level view needs a `key` property, I suggest using the index of the array. So `(token) =>` would become `(token, index) =>`

Comment (willbach) [reply packages/ui/dialogs/my-account/index.tsx]: Also, line 200 of `my-account` should be changed to `const panelHeights = [150, 250, 60, 60, 60, 60, 140, 60, 190, 180, 60, 120]` (it should be of length 12 now that you've removed one of the options)

File: packages/lndr/erc20-utils/index.ts (hunk 1 above)
Comment (arutun) [reply packages/lndr/erc20-utils/index.ts]: Fixed

File: packages/ui/dialogs/my-account/index.tsx (hunk 2 above)
Comment (arutun) [reply packages/ui/dialogs/my-account/index.tsx]: good find

**Pull request merged**